README.md

```diff
@@ -0,0 +1,35 @@
+# Mini API Testing Project
+## Description
+This project is to test data that is retrieved from the Open Weather API (Application Programming Interface) and compare it to a JSON file which has all the cities in the world. This checks to see if it has passed or failed.
```

Comment:
We are not comparing the data from the API with the JSON data. The JSON data is there to help you generate a random City ID